google/cloud/ndb/tasklets.py

```diff
@@ -53,8 +53,8 @@ def main():
 return a value from a generator in Python 3, a `StopIteration`
 exception is raised with the return value as its argument. The event
 loop catches the exception and uses the exception argument as the
-result of the tasklet.  This won't work for Python 2. If you need to
-support Python 2, as the library itself does, you'll need to raise a
+result of the tasklet. This won't work for Python 2. If you need to
+support Python 2, you'll need to raise a
```

Comment:
This bit can be removed since we're dropping python 2 : `If you need to support Python 2`